eZ/Publish/Core/Repository/Tests/Values/ContentType/FieldDefinitionCollectionTest.php

```diff
@@ -0,0 +1,318 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\Repository\Tests\Values\ContentType;
+
+use Closure;
+use eZ\Publish\API\Repository\Values\ContentType\FieldDefinition as APIFieldDefinition;
+use eZ\Publish\Core\Repository\Values\ContentType\FieldDefinition;
+use eZ\Publish\Core\Repository\Values\ContentType\FieldDefinitionCollection;
+use PHPUnit\Framework\TestCase;
+
+final class FieldDefinitionCollectionTest extends TestCase
+{
+    public function testGet(): void
```

Comment:
could you add `@covers` to all those methods? 

Not insisting, because we do not use it right now, but it might come in handy at some point (I also expect it's beneficial IDE go to test nav - just guessing). 

e.g.:
``` php
/**
 * @covers \eZ\Publish\API\Repository\Values\ContentType\FieldDefinitionCollection::get
 */
```
(intentionally no braces after the method name)